docs/src/tutorials/object_detection.md

```diff
@@ -6,16 +6,16 @@ a person classifier and then use this classifier with a sliding window to identi
 The key challenge in creating a classifier is that it needs to work with variations in illumination, pose and occlusions in the image. To achieve this, we will train
 the classifier on an intermediate representation of the image instead of the pixel-based representation. Our ideal representation (commonly called feature vector)
 captures information which is useful for classification but is invariant to small changes in illumination and occlusions. HOG descriptor is a gradient-based
-representation which is invariant to local geometric and photometric changes (i.e. shape and illumination changes) and so is a good choice for our problem. In fact
-HOG descriptors are widely used for object detection.
+representation which is invariant to local geometric and photometric changes (i.e. shape and illumination changes) and so is a good choice for our problem. In fact HOG descriptors are widely used for object detection.
 
-We will use [this data](https://drive.google.com/file/d/0B9V0KF3ZHWtWa0hGNHJDSm1XZnM/view?usp=sharing) for training our classifier. We will start by loading the data and computing HOG features of all the images.
+Run the the script get_data.bash provided [here](https://drive.google.com/open?id=0B9V0KF3ZHWtWR1dBR2VZUDctUGc) (Change the variables `path_to_tutorial` in preprocess.jl and path to julia executable in get_data.bash). This script will download the required datasets. We will start by loading the data and computing HOG features of all the images.
```

Comment:
Is that Google drive repository likely to be permanent? Maybe also say that the whole thing needs to be downloaded and decompressed, and then run `get_data.bash`.

Too bad that script will only work on Unix, but ATM I don't see a great way to decompress a `.tar.gz` purely within Julia.

Reply:
The drive repository should be permanent. I have updated the part and explained that the file needs to be downloaded and decompressed too.